The /package.json and /test/test-index.js files point to missing /index.js

Looks like the [test/test-index.js](https://github.com/mozilla/universal-search-addon/blob/master/test/test-index.js) may just be boilerplate, but it seems to be failing for me with the following vomit:

```sh
➜  test git:(master) pwd
/Users/pdehaan/dev/github/universal-search-addon/test

➜  test git:(master) node test-index.js

module.js:340
    throw err;
          ^
Error: Cannot find module '../'
    at Function.Module._resolveFilename (module.js:338:15)
    at Function.Module._load (module.js:280:25)
    at Module.require (module.js:364:17)
    at require (module.js:380:17)
    at Object.<anonymous> (/Users/pdehaan/dev/github/universal-search-addon/test/test-index.js:1:74)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)

➜  test git:(master) echo $?
8
```

Not sure if I'm missing some piece of setup, or if it was able to resolve `../`, what it'd then do with this line (or if I need to install some other prerequisites):
```js
require("sdk/test").run(exports);
```

--- a/package.json
+++ b/package.json
@@ -26,7 +26,6 @@
   },
   "homepage": "https://github.com/mozilla/universal-search-addon",
   "license": "MPL-2.0",
-  "main": "index.js",
   "repository": {
     "type": "git",
     "url": "git+https://github.com/mozilla/universal-search-addon.git"

